src/main/java/at/hannibal2/skyhanni/data/SackAPI.kt

```diff
@@ -259,7 +260,13 @@ object SackAPI {
         updateSacks(sackEvent)
         sackEvent.postAndCatch()
         if (chatConfig.hideSacksChange) {
-            event.blockedReason = "sacks_change"
+            if (!chatConfig.onlyHideSacksChangeOnGarden) {
+                event.blockedReason = "sacks_change"
+            } else {
+                if (LorenzUtils.skyBlockIsland == IslandType.GARDEN) {
+                    event.blockedReason = "sacks_change"
+                }
+            }
```

Comment:
```suggestion
            if (chatConfig.hideSacksChange && (!chatConfig.onlyHideSacksChangeOnGarden || IslandType.GARDEN..isInIsland())) {
    		event.blockedReason = "sacks_change"
			}
```

This can be turned into a single if statement that also uses the correct functions